### DISCUSSION TITLE
Support for Asus Vivostick?

### DISCUSSION THREAD

**Discussion by vetko** [dominiksalvet/asus-fan-control#90] [Q&A]
Hi,

I am looking forward to installing linux on an **ASUS VivoStick TS10-B189D** (Intel Atom x5-Z8350, 2GB, eMMC).
According to some 1-2 years old reports, the fan on this device runs 100% all the time while running linux on it, even if the device is not under any load.

Do you happen to have any experience with this / similar Asus devices?
I am wondering what can I expect for if I try to use asus-fan-control on this stick.

Thank you very much!

**Comment by dominiksalvet** [comment]
Well, as far as weird AFC things go, there were some attempts to make AFC run on a PC with an ASUS motherboard but I do not know how it turned out... Certainly, it would be very exciting to see AFC running on such a device but I do not have high hopes for that. Worth the try, though. Make sure you will let us know! :rocket:
